src/AcceptanceTests/Receiving/Issue_532_When_incoming_message_lock_token_is_lost_in_send_atomic_with_receive_mode.cs

```diff
@@ -0,0 +1,100 @@
+namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using MyNamespace;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+    using AcceptanceTesting.Customization;
+
+    public class Issue_532_When_incoming_message_lock_token_is_lost_in_send_atomic_with_receive_mode : NServiceBusAcceptanceTest
+    {
+        const int LockDurationOnIncomingMessageInSeconds = 5;
+
+        [Test]
+        public async Task Should_not_dispatch_outgoing_message()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Sender>(builder => builder.When((session, ctx) => session.SendLocal(new InitialMessage())))
+                .WithEndpoint<Receiver>()
+                .Done(ctx => ctx.TimesDispatchedMessageSent > 1)
+                .Run();
+            
+            Assert.That(context.TimesDispatchedMessageReceived, Is.Zero);
+        }
+
+        public class Context : ScenarioContext
+        {
+            public int TimesDispatchedMessageReceived { get; set; }
+            public int TimesDispatchedMessageSent { get; set; }
+        }
+
+        public class Sender : EndpointConfigurationBuilder
+        {
+            public Sender()
+            {
+                EndpointSetup<DefaultServer>(config =>
+                {
+                    var transport = config.UseTransport<AzureServiceBusTransport>();
+                    transport.Queues().LockDuration(TimeSpan.FromSeconds(LockDurationOnIncomingMessageInSeconds));
+                    transport.Queues().MaxDeliveryCount(100);
+                    transport.MessageReceivers().AutoRenewTimeout(TimeSpan.Zero);
```

Comment:
Out of scope: Interesting so in order to disable auto renew we have to set the timespan to zero. So there is no disable auto renew extension method?